runtime/moonbase/src/xcm_config.rs

```diff
@@ -376,7 +376,7 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type PriceForSiblingDelivery = polkadot_runtime_common::xcm_sender::NoPriceForMessageDelivery<
 		cumulus_primitives_core::ParaId,
 	>;
-	type MaxActiveOutboundChannels = ConstU32<128>;
+	type MaxActiveOutboundChannels = MessageQueueHeapSize;
```

Comment:
You confused `MaxPageSize` with `MaxActiveOutboundChannels`. This change is wrong